### PR TITLE
ci: only use Python 3.10+

### PR DESCRIPTION
**Pull request by lengau**
This was already happening thanks to tox changes, but this does do unit tests on 3.13 on focal now too.

- [ ] Have you signed the [CLA](http://www.ubuntu.com/legal/contributors/)?

-----

### CODE REVIEW

**Review by tigarmo — APPROVED** [review]

**Comment by tigarmo** [comment]
Please fix the "unly" on the commit message (😆)

**Review by tigarmo — APPROVED** [review]

File: .github/workflows/tests.yaml [hunk 1]
```diff
@@ -136,7 +124,7 @@ jobs:
           - adjective: jammy
           - os: ubuntu-20.04
             python-version: "3.10"
-            adjective: jammy
+            adjective: focal
```
Comment (cmatsuoka) [inline .github/workflows/tests.yaml:127]: Good catch!

**Review by cmatsuoka — APPROVED** [review]

**Comment by cmatsuoka** [comment]
I think we'll need @mr-cal's assistance to fix the required tests list in order to land this.

**Comment by lengau** [comment]
@cmatsuoka could you change the required ci items to match my typo fix?

**Pull request merged**